sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/HttpClientBuilderWithTimeoutExample.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.core.util.Context;
+import com.azure.core.util.HttpClientOptions;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Locale;
+
+/**
+ * This example shows how to start using the Azure Storage Blob SDK for Java.
```

Comment:
This should be more descriptive and include that it shows how to use read/write/connection level timeouts.